src/Api/Models/Request/Organizations/OrganizationSsoRequestModel.cs

```diff
@@ -37,6 +37,11 @@ public SsoConfig ToSsoConfig(SsoConfig existingConfig)
 
     public class SsoConfigurationDataRequest : IValidatableObject
     {
+        private static readonly Regex _certificateRegex = new Regex(@"(((BEGIN|END) CERTIFICATE)|([\-\n\r\t\s\f]))",
```

Comment:
Having this here is good, not sure we should be adding `RegexOptions.Compiled` however.